source/classes.tex

```diff
@@ -2402,9 +2402,9 @@
 \end{note}
 
 \pnum
+\begin{note}
 A non-explicit copy/move constructor\iref{class.copy.ctor} is
 a converting constructor.
-\begin{note}
 An implicitly-declared copy/move constructor is not an explicit constructor;
 it can be called for implicit type conversions.
```

Comment:
This is now pretty heavily redundant.  How about

> A non-explicit copy/move constructor&nbsp;([class.copy.ctor]), including one implicitly declared, is a converting constructor.

Reply:
That's nice, thanks!

Reply:
@Eisenwave ^^^ could you please have a look?

Reply:
Yeah I'll go through all the things I have to do in this repo tomorrow.

Reply:
@Eisenwave Can you please send that as a separate commit?